socorro/processor/rules/mozilla.py

```diff
@@ -627,15 +652,15 @@ def action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
 
 class TopMostFilesRule(Rule):
     """Origninating from Bug 519703, the topmost_filenames was specified as
-    singular, there would be only one.  The original programmer, in the
+    singular, there would be only one. The original programmer, in the
     source code stated "Lets build in some flex" and allowed the field to
-    have more than one in a list.  However, in all the years that this existed
-    it was never expanded to use more than just one.  Meanwhile, the code
+    have more than one in a list. However, in all the years that this existed
+    it was never expanded to use more than just one. Meanwhile, the code
     ambiguously would sometimes give this as as single value and other times
     return it as a list of one item.
 
     This rule does not try to reproduce that imbiguity and avoids the list
-    entirely, just giving one single value.  The fact that the destination
+    entirely, just giving one single value. The fact that the destination
```

Comment:
I'm not sure what made these changes. I don't think I did it by hand, so maybe it was something automated?